app/src/app.config.json

```diff
@@ -19,7 +19,7 @@
     "clientId": "alfresco",
     "scope": "openid",
     "secret": "",
-    "implicitFlow": true,
+    "codeFlow": true,
```

Comment:
this is a breaking change? you could probably set "implicitFlow" to "false" and use "codeFlow" alongside